### DISCUSSION TITLE
Ошибка. Select по системным таблицам

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#443] [ОШБК!] [answered]
Пытаюсь сделать селект по сист таблицам, а оно выдаёт ошибку. Подскажите что не так?

![Screenshot_3](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113771444/e47e4583-8325-41e5-bd66-980fb071f18f)

**Comment by poluninatm** [comment, marked as answer]
Ошибку выдает база данных, которой не нравится название "table" в качестве добавляемого поля. Оно было в исключенных, но легаси вынудило убрать проверку в этом месте. С указанным селектом это никак не связано

**Reply by mihail-cornev** [reply]
я понял свою ошибку, спасибо.
